client/coral-framework/actions/asset.js

```diff
@@ -10,30 +10,30 @@ export const fetchAssetRequest = () => ({type: actions.FETCH_ASSET_REQUEST});
 export const fetchAssetSuccess = asset => ({type: actions.FETCH_ASSET_SUCCESS, asset});
 export const fetchAssetFailure = error => ({type: actions.FETCH_ASSET_FAILURE, error});
 
-const updateConfigRequest = () => ({type: actions.UPDATE_CONFIG_REQUEST});
-const updateConfigSuccess = config => ({type: actions.UPDATE_CONFIG_SUCCESS, config});
-const updateConfigFailure = () => ({type: actions.UPDATE_CONFIG_FAILURE});
+const updateAssetSettingsRequest = () => ({type: actions.UPDATE_ASSET_SETTINGS_REQUEST});
```

Comment:
Great initiative to keep naming conventions clean. Things like this make all the difference toward readability.